action.yml

```
@@ -1,4 +1,4 @@
-name: 'Build and publish Docker Image to GitHub Registry'
+name: 'Build and publish Docker Image to GitHub Registry with custom Dockerfile'
 description: 'GitHub Action to publish Docker Images to GitHub Registry'
```

I guess it's better to keep our Action's name as simple as possbile. But we indeed can update Action's desctription a bit.
```suggestion
name: 'Build and publish Docker Image to GitHub Registry'
description: 'GitHub Action to publish Docker Images to GitHub Registry. It's possible to specify custom Dockerfile and\or build context, if it's required.'
```